topics/phylogenetics/tutorials/MTB phylogeny/tutorial.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: tutorial_hands_on
+title: Tree thinking for tuberculosis evolution and epidemiology
+zenodo_link: 'https://zenodo.org/record/6010176'
+tags:
+  - prokaryotes
+  - phylogenetics
+  - evolution
+level: Beginner
+objectives:
+- Understand the basic concepts behind phylogenetic trees
+- Be able to read and interrogate a phylogeny encountered in the literature
+time_estimation: 1H
+contributors:
+- Christoph Stritt
+- Daniela Brites
+- Galo A. Goig
```

Comment:
these names should be references to the CONTRIBUTORS.yaml file, you'll need to add them in there.

Reply:
names were added